### PR TITLE
Update metadata.md

### PR DESCRIPTION
**Pull request by schwenk102**

### CODE REVIEW

File: context/app/markdown/docs/metadata.md [hunk 1]
```diff
@@ -1,39 +1,51 @@
-# Metadata
+## Metadata
```
Comment (mccalluc) [inline context/app/markdown/docs/metadata.md:1]: Keeping the main title with one `#` would be better.

```suggestion
# Metadata
```

File: context/app/markdown/docs/metadata.md [hunk 2]
```diff
@@ -1,39 +1,51 @@
...
 
 HuBMAP metadata documentation and submission specifications
 
-## Donor
+**Donor**
```
Comment (mccalluc) [inline context/app/markdown/docs/metadata.md:5]: No -`##` is better syntax for headings.

File: context/app/markdown/docs/metadata.md [hunk 3]
```diff
@@ -1,39 +1,51 @@
...
-|TMT LC-MS (proteomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Lihua Jiang](mailto:lihuaj@stanford.edu)|
-|Untargeted LC-MS (eg: LC-MS/MS-proteomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)|
-|Untargeted LC-MS (metabolomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Kevin Contrepois](mailto:kcontrep@stanford.edu)|
-|WGS|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/wgs)|[Aaron Horning](mailto:ahorning@stanford.edu)|
+**Assay**
+
+Assay|Document|Specification|Contact
+|--|--|--|--
+|AF (Autofluorescence)|[Document](https://github.com/hubmapconsortium/portal-ui/blob/master/context/app/markdown/docs/assay/af.md)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/af)|[Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)
```
Comment (mccalluc) [inline context/app/markdown/docs/metadata.md:22]: No - The original had line-end pipes, and now they're missing.

File: context/app/markdown/docs/metadata.md (hunk 2 above)
Comment (schwenk102) [reply context/app/markdown/docs/metadata.md:5]: @ngehlenborg @mccalluc Please go ahead with these changes

File: context/app/markdown/docs/metadata.md (hunk 3 above)
Comment (schwenk102) [reply context/app/markdown/docs/metadata.md:22]: I do not know how to make the pipe change here. if you can add that in, that would be great! @mccalluc @ngehlenborg

**Comment by mccalluc** [comment]
@schwenk102 : There are now conflicts between this file and master. It needs to be the responsibility of the author to resolve conflicts.

**Comment by mccalluc** [comment]
Closing, as discussed on slack.

**Pull request closed without merging**